source/openpower/volumes.rst

```diff
@@ -0,0 +1,32 @@
+Interacting with Volumes on OpenPOWER
+=====================================
+
+Each Operating System interacts with the volumes attached to it in various ways.
+The most common ways to interact with volumes are through packaged programs such
+as ``fdisk``, ``mount``/``umount``, and other LVM commands. Alongside these packages,
+there are other packages that are necessary to view and allow the system to properly
+interact with the volumes that the user is attempting to manipulate.
+
+Volumes on ppc64
+--------------------
+
+One of the most crucial packages for proper POWER volume management is ``ppc64-diag``.
```

Comment:
"actually being able to use volumes on POWER platform" sounds better than "proper POWER volume management"